jumpstarter/states.py

```diff
@@ -39,22 +55,42 @@ def __init__(self, actor_state=ActorState):
             send_event=True,
         )
 
+        # TODO: Does transitions have an API for Enum based parallel?
```

Comment:
It currently doesn't, unfortunately. If you can come up with an API, we can propose it to the transitions project.

Reply:
Would be happy to.